### ISSUE TITLE
Systematic testing capability

### ISSUE THREAD

**Issue by amartinhuertas** [GiteonCaulfied/COMP4560_stokes_ml_project#2]
Let me open a different issue to have specialized discussions in regards to this. 

Let me start by copying and pasting @GiteonCaulfied comment in the other issue.

>Hi @amartinhuertas @rhyshawkins

> I've implemented a naive systematic testing method, including a new notebook Systematic_testing.ipynb and a text file ModelList.txt that contains a list of different hyperparameters settings for the specific result_20k_zero dataset (including hidden layers neurons, activation function, Loss function type, epoch, batch size and learning rate)

> By running the Systematic_testing.ipynb, it will read each line of the ModelList.txt file to build a NN model that aligns with the settings specified in that line. After that, the plot of training loss and validation loss during training, also the plot of the Best/Worst case input and output for training set and testing set will be generated when the code is running.

> One thing about this naive systematic testing method is that these plots for different models are not put together in a complete grid structure after each model has been trained and tested. It also can't be selected and viewed with a drop down box in an interactive way like the one plot in the given Julia notebook. Therefore, to compare the performance of each model, one may need to keep scrolling down and scrolling up and that can be kind of inconvenient. I'll try to think of some ways to make it better if possible.

> The updated version of code will be uploaded shortly after this comment as usual.
